src/useGoogleAnalytics.ts

```diff
@@ -28,13 +40,13 @@ if (googleAnalyticsId && !initialized) {
         GoogleAnalytics.initialize([{
             trackingId: googleAnalyticsId,
             gaOptions: {
-              cookieFlags: 'SameSite=None; Secure',
-              testMode: debug ? true : false,
-              ...(nonce && { nonce: nonce }) 
+                cookieFlags: 'SameSite=None; Secure',
```

Comment:
Nitpick: I feel `SameSite=None; Secure` could be made constants if in future there are more cookie attributes added like `Partitioned` in future.